### PR TITLE
[REACTOS] Fix '.  \n' and similar typos

### PR DESCRIPTION
**Pull request by SergeGautherie**

### CODE REVIEW

File: drivers/usb/usbport/pnp.c [hunk 1]
```diff
@@ -325,7 +325,7 @@ USBPORT_QueryPciBusInterface(IN PDEVICE_OBJECT FdoDevice)
     PIRP Irp;
     NTSTATUS Status;
 
-    DPRINT("USBPORT_QueryPciBusInterface: ...  \n");
+    DPRINT("USBPORT_QueryPciBusInterface: ...\n");
```
Comment (Extravert-ir) [inline drivers/usb/usbport/pnp.c]: Remove this change

File: hal/halx86/mp/ioapic.c [hunk 2]
```diff
@@ -603,8 +603,7 @@ VOID IOAPICDump(VOID)
       }
 
       DbgPrint(".... IRQ redirection table:\n");
-      DbgPrint(" NR Log Phy Mask Trig IRR Pol"
-	       " Stat Dest Deli Vect:   \n");
+      DbgPrint(" NR Log Phy Mask Trig IRR Pol Stat Dest Deli Vect:\n");
```
Comment (Extravert-ir) [inline hal/halx86/mp/ioapic.c]: And this one as well

File: drivers/usb/usbport/pnp.c (hunk 1 above)
Comment (SergeGautherie) [reply drivers/usb/usbport/pnp.c]: As you wish :-/

File: hal/halx86/mp/ioapic.c (hunk 2 above)
Comment (SergeGautherie) [reply hal/halx86/mp/ioapic.c]: As you wish :-/

**Comment by SergeGautherie** [comment]
> remove all stuff related to code changes

For potential follow-up PRs, does that apply to debug log only, or to actual user interface too?

**Pull request merged**